docs/config-file-package-configuration.yml.md

```diff
@@ -0,0 +1,99 @@
+# The package configuration file (Experimental)
+
+A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
+provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
+opposed to packages.
+
+## Specifying the target package and provenance
+
+Each package configuration applies exactly to one package-Id and provenance which must be specified. The provenance
+can either specify a `source artifact` or a `VCS location and revision`. Thus in case one scans the `source artifact`
```

Comment:
Backticks are usually used for code, what about using *italics*  instead?

Reply:
I turned several (not all) backticks in this file into italics.